src/plugins/intel_cpu/src/transformations/transformation_pipeline.cpp

```diff
@@ -1267,7 +1265,11 @@ void Transformations::MainSnippets() {
          ov::intel_cpu::brgemm_utils::is_fp16_supported());
     const bool isMHASupported = !is_LLM && is_infer_prc_supported_by_brgemm;
 #else
-    const bool isMHASupported = false;
+    const auto is_infer_prc_supported_by_brgemm =
```

Comment:
Reminder: please reuse this variable in `isMlpSeqSupported` after PR#31754 is merged into master